java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -256,7 +256,7 @@ private boolean isConstraintFromObjectDomain(@Nullable Constraint constraint) {
       return constraint instanceof ObjectConstraint;
     }
 
-    private List<JavaFileScannerContext.Location> flowForNullableMethodParameters(ExplodedGraph.Node node) {
+    private List<JavaFileScannerContext.Location> flowForNullableMethodParametersOrFinalField(ExplodedGraph.Node node) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 17 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3776)